pkg/cri/server/image_pull.go

```diff
@@ -194,8 +209,14 @@ func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest)
 		}
 	}
 
-	log.G(ctx).Debugf("Pulled image %q with image id %q, repo tag %q, repo digest %q", imageRef, imageID,
-		repoTag, repoDigest)
+	imageNamed, _ := distribution.ParseNamed(ref)
```

Comment:
Error should be handled

Reply:
OK, if there is an error, we should return "", I think.
Updated.